deltachat-ios/Controller/QrPageController.swift

```diff
@@ -33,6 +33,12 @@ class QrPageController: UIPageViewController, ProgressAlertHandler {
     // MARK: - lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        // temp fix - ToDo: check if this is needed for >iOS13 devices
+        if #available(iOS 13, *) {}
```

Comment:
lint is complaining here. can you put the else into the same line?

Reply:
The code block became obsolete with lastest master